docs/en/enterprise-edition/content-collections/application-security/application-security.adoc

```diff
@@ -1,7 +1,58 @@
+The Application Security on Prisma Cloud console provides comprehensive security for your software engineering environment, covering the entire software development lifecycle from code to cloud.
 
+=== Scanning Capabilities
+
+The Application Security comprehensive scanning capabilities include: 
+
+* *Infrastructure as Code (IaC)*: Identifies misconfigurations and vulnerabilities within your IaC templates (e.g., Terraform, CloudFormation) to ensure secure cloud infrastructure provisioning
+
+* *Software Composition Analysis (SCA)*: Scans for known vulnerabilities in your application dependencies (direct and indirect packages) to mitigate potential security risks from third-party components
+
+* *License* noncompliance: Scans for software license noncompliance
+
+* *Secrets Security*: Detects exposed secrets like passwords, API keys, and tokens within your code to prevent unauthorized access and breaches
+
+* *CI/CD Pipeline Security*: Analyzes your CI/CD pipelines for potential security weaknesses, ensuring a secure software delivery process
+
+* *Container Image scanning*: Scans container images for vulnerabilities, misconfigurations, and malware to secure your containerized applications
+
+==== Seamless Integration
+
+Prisma Cloud leverages existing workflows by integrating with your development environment through:
+
+* *Web UI*: The central platform for initiating scans, managing findings, and configuring security policies
+
+* *Integrated Development Environment (IDE)*: Provides security feedback directly within your development environment
+
+* *Command Line Interface (CLI)*: Enables scripting and automation of security tasks
+Integrated Development Environment (IDE): Provides security feedback directly within your development environment
+
+* *Version Control System (VCS)*: Integrates with your version control system to scan default or non-default branches containing open Pull Requests (PR) or Merge Requests (MR). In addition the system also performs periodic scans on specific branches within each repository.  
+
+=== Visibility and Insights
+
+* *Technical DNA*: Maps your software engineering ecosystem, including programming languages, frameworks, CI/CD pipelines, and plugins. This comprehensive overview allows for a deeper understanding of your development environment
+
+* *Supply Chain Security*: Provides an inventory and visualization of application dependencies, offering a clear view of your software's supply chain
```

Comment:
I'm not sure we should call this "Supply Chain Security" - that usually implies software integrity validation. Did you confirm this with the team?